Hunting Queries/MultipleDataSources/SQLAlertCorrelationwithCommonSecurityLogsandAuditLogs.yaml

```diff
@@ -0,0 +1,52 @@
+id: dc5adcc9-70ab-4fba-8690-f57767e8ca02
+name: SQL Alert Correlation with CommonSecurityLogs and AuditLogs
+description: |
+  'This query combines different SQL alerts with CommonSecurityLogs and AuditLogs helping analysts /investigate any possible SQL related attacks faster
+   thus reducing MTTR'
+requiredDataConnectors:
+  - connectorId: PaloAltoNetworks
+    dataTypes:
+      - CommonSecurityLog
+  - connectorId: AzureSecurityCenter
+    dataTypes:
+      - SecurityAlert (ASC)
+  - connectorId: AzureActiveDirectory
+    dataTypes:
+      - AuditLogs
+tactics:
+  - InitialAccess
+  - Impact
+relevantTechniques:
+  - T1190
+  - T1078
+query: |
+ let timeframe = 1d;
```

Comment:
We've moved to not expressing a timeframe in hunting queries where possible. That allows the user to select their own timeframe in the UI. If you just leave out all timeframe elements it will base it off that UI timeframe instead.